README.md

```diff
@@ -341,3 +341,102 @@ try {
     echo 'Exception when calling SellersApi->getMarketplaceParticipations: ', $e->getMessage(), PHP_EOL;
 }
 ```
+
+## Custom Authorization Signer
+There is a possibility to use custom authorization signer.
+This may be useful if you want to have more control on adding "Authorization" signature to request. 
+
+```php
+// CustomAuthorizationSigner.php
+use GuzzleHttp\Psr7\Request;
+use SellingPartnerApi\Contract\AuthorizationSignerContract;
+
+class CustomAuthorizationSigner implements AuthorizationSignerContract
+{
+    public function sign(Request $request, Credentials $credentials): Request
+    {
+        // Calculate request signature and request date.
+        
+        $requestDate = '20220426T202300Z';
+        $signatureHeaderValue = 'some calculated signature value';
+        
+        $signedRequest = $request
+            ->withHeader('Authorization', $signatureHeaderValue)
+            ->withHeader('x-amz-date', $requestDate);
+        
+        return $signedRequest;
+    }
+
+    // ...
+}
+
+// Consumer code
+<?php
+require_once(__DIR__ . '/vendor/autoload.php');
+
+use SellingPartnerApi\Api;
+use SellingPartnerApi\Configuration;
+use SellingPartnerApi\Endpoint;
+use CustomAuthorizationSigner;
+
+$config = new Configuration([
+    ..., 
+    'authorizationSigner' => new CustomAuthorizationSigner(),
+]);
+$api = new Api\SellersApi($config);
+try {
+    $result = $api->getMarketplaceParticipations();
+    $headers = $result->getHeaders();
+    print_r($headers);
+} catch (Exception $e) {
+    echo 'Exception when calling SellersApi->getMarketplaceParticipations: ', $e->getMessage(), PHP_EOL;
+}
+
+```
+
+## Custom Request Signer
+More over there is a possibility to customize whole request signing process.
+It may be needed if you need to do additional check during the signing of request.
+Or you need to proceed request signing on separate service instance.
```

Comment:
Please change to:

You may also need to customize the entire request signing process -- for instance, if you need to call an external service in the process of signing the request. You can do so by creating an implementation of the `RequestSigner` interface, and passing an instance of it into the `Configuration` constructor array.

Reply:
Done.